electron-{source,bin,chromedriver}: updates, electron-source_39: fix build

teutat3s opened the pull request:
Changelog: https://github.com/electron/electron/releases/tag/v37.10.0
Diff: https://github.com/electron/electron/compare/refs/tags/v37.9.0...v37.10.0

Changelog: https://github.com/electron/electron/releases/tag/v38.7.0
Diff: https://github.com/electron/electron/compare/refs/tags/v38.6.0...v38.7.0

Changelog: https://github.com/electron/electron/releases/tag/v39.1.2
Diff: https://github.com/electron/electron/compare/refs/tags/v39.1.1...v39.1.2
Changelog: https://github.com/electron/electron/releases/tag/v39.2.0
Diff: https://github.com/electron/electron/compare/refs/tags/v39.1.2...v39.2.0

Fixes build of `electron_39` that was mentioned in https://github.com/NixOS/nixpkgs/pull/459455#issuecomment-3518742383, breaking change was in commit https://github.com/NixOS/nixpkgs/pull/459455/commits/6236808dd4afde2b99d8ff0424e661d1ed1cfe8f.

ZHF: #457852

Fixes https://github.com/NixOS/nixpkgs/issues/461712

<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

teutat3s committed:
electron-source.electron_39: 39.1.1 -> 39.1.2

- Changelog: https://github.com/electron/electron/releases/tag/v39.1.2
- Diff: https://github.com/electron/electron/compare/refs/tags/v39.1.1...v39.1.2

teutat3s committed:
electron_39-bin: 39.1.1 -> 39.1.2

- Changelog: https://github.com/electron/electron/releases/tag/v39.1.2
- Diff: https://github.com/electron/electron/compare/refs/tags/v39.1.1...v39.1.2

teutat3s committed:
electron-chromedriver_39: 39.1.1 -> 39.1.2

- Changelog: https://github.com/electron/electron/releases/tag/v39.1.2
- Diff: https://github.com/electron/electron/compare/refs/tags/v39.1.1...v39.1.2

yayayayaka commented:
Building on x86_64-linux

teutat3s committed:
electron_37-bin: 37.9.0 -> 37.10.0

- Changelog: https://github.com/electron/electron/releases/tag/v37.10.0
- Diff: https://github.com/electron/electron/compare/refs/tags/v37.9.0...v37.10.0

teutat3s committed:
electron-chromedriver_37: 37.9.0 -> 37.10.0

- Changelog: https://github.com/electron/electron/releases/tag/v37.10.0
- Diff: https://github.com/electron/electron/compare/refs/tags/v37.9.0...v37.10.0

teutat3s committed:
electron_38-bin: 38.6.0 -> 38.7.0

- Changelog: https://github.com/electron/electron/releases/tag/v38.7.0
- Diff: https://github.com/electron/electron/compare/refs/tags/v38.6.0...v38.7.0

teutat3s committed:
electron-chromedriver_38: 38.6.0 -> 38.7.0

- Changelog: https://github.com/electron/electron/releases/tag/v38.7.0
- Diff: https://github.com/electron/electron/compare/refs/tags/v38.6.0...v38.7.0

teutat3s committed:
electron_39-bin: 39.1.2 -> 39.2.0

- Changelog: https://github.com/electron/electron/releases/tag/v39.2.0
- Diff: https://github.com/electron/electron/compare/refs/tags/v39.1.2...v39.2.0

teutat3s committed:
electron-chromedriver_39: 39.1.2 -> 39.2.0

- Changelog: https://github.com/electron/electron/releases/tag/v39.2.0
- Diff: https://github.com/electron/electron/compare/refs/tags/v39.1.2...v39.2.0

teutat3s committed:
electron-source.electron_37: 37.9.0 -> 37.10.0

- Changelog: https://github.com/electron/electron/releases/tag/v37.10.0
- Diff: https://github.com/electron/electron/compare/refs/tags/v37.9.0...v37.10.0

teutat3s committed:
electron-source.electron_38: 38.6.0 -> 38.7.0

- Changelog: https://github.com/electron/electron/releases/tag/v38.7.0
- Diff: https://github.com/electron/electron/compare/refs/tags/v38.6.0...v38.7.0

yuyuyureka commented:
How are you planning to fix the build of electron_39? It is broken, because an outdated gclient2nix / NIX_PATH was used when running the update script (it must include https://github.com/NixOS/nixpkgs/commit/4dc9b83879ce51e180f37acb8e78ededdbf72798).

yuyuyureka reviewed (dismissed):
You first fixed electron_39 build, and then broke it again in 5bcfc655f07e1ca4e7db969044cb8b58454810f3

teutat3s commented:
Yep, fixed again.

teutat3s committed:
electron-source.electron_39: 39.1.2 -> 39.2.0

- Changelog: https://github.com/electron/electron/releases/tag/v39.2.0
- Diff: https://github.com/electron/electron/compare/refs/tags/v39.1.2...v39.2.0

leona-ya reviewed (approved):
`electron_37`, `electron_38`, `electron_39` now build fine on x86_64-linux.
